content/general/authentication.textile

```diff
@@ -7,68 +7,29 @@ languages:
     - Token authentication
+    - Selecting an authentication mechanism#selecting-auth
     - Capabilities explained
     - Capability operations
     - Understanding Identified clients#identified-clients
-    - Tokens
+    - Token Types#tokens
 ---
 
 Both the "REST client library":/rest/authentication and the "Realtime client library":/realtime/authentication use common authentication mechanisms. The two schemes supported by Ably are "Basic Authentication":#basic-authentication, which uses your private "API key":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key, and "Token Authentication":#token-authentication, which uses short-lived tokens for access which are periodically renewed. 
 
 Different token-issuing mechanisms can be used with Ably; the default is to use "Ably Tokens":#tokens which you request from Ably based on an Ably "TokenRequest":/realtime/authentication#token-request that you sign and issue from your servers to clients; or a "JSON Web Token":https://jwt.io (JWT) which you generate on your servers and sign using your private API key. Token Authentication, in most cases, is the recommended strategy on the client-side as it provides more fine-grained access control and limits the risk of exposed or compromised credentials.
 
-h2(#intro). Quick intro to Ably's Authentication schemes
-
-**Using Basic Authentication**
-Every application has one or more private API keys that you set up and "configure via the dashboard":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys. It is recommended that private API keys are never shared with untrusted parties, and as such, should only be used by your trusted secure servers when authenticating with Ably. In the example below, a server is publishing a message using the Ably REST library over HTTP, which in turn is using an API key for Basic Authentication.
-
-<a href="/images/diagrams/auth-intro-server.png" target="_blank">
-  <img src="/images/diagrams/auth-intro-server.png" style="width: 100%" alt="Server-side basic auth diagram">
-</a>
-
-**Using Token Authentication**
-Client-side devices should generally be considered untrusted, and as such, it is important that you minimize the impact of any credentials being compromised on those devices. Therefore, the recommended strategy for authenticating client-side devices is to issue them with time-limited tokens with only the privileges (capabilities) that are needed for that device. It is therefore the responsibility of the developer to identify each device before issuing a token. In the example below, we show you a typical token authentication workflow for client-side devices.
-
-<a href="/images/diagrams/auth-intro-client-side.png" target="_blank">
-  <img src="/images/diagrams/auth-intro-client-side.png" style="width: 100%" alt="Client-side basic auth diagram">
-</a>
-
-h3(#tutorials). Tutorials &amp; Examples
+h2(#tutorials). Tutorials &amp; Examples
```

Comment:
I think the tutorials header should be h3, so it doesn't get counted as a separate heading but as part of the intro in this page, only with a little more focus. What do you think?